lib/modules/commentTools.js

```diff
@@ -672,8 +674,16 @@ function addButtonToMacroGroup(groupName, button) {
 	group.dropdown.append($('<li>').append(button));
 }
 
+function getDebugMacros() {
+	if (!['Enhancement', 'RESissues'].includes(currentSubreddit())) return [];
```

Comment:
I could imagine someone changing `currentSubreddit()` to return lowercase (or _maybe_ there are situations where the URL isn't canonicalized); might as well make this `isCurrentSubreddit('Enhancement') || isCurrentSubreddit('RESissues')`, or make `isCurrentSubreddit` variadic like `isPageType`